src/OmniSharp.Abstractions/Options/CodeActionOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OmniSharp.Options
+{
+    public class CodeActionOptions
+    {
+        public string[] LocationPaths { get; set; }
+
+        public IEnumerable<string> GetLocations(IOmniSharpEnvironment env)
+        {
+            if (LocationPaths == null) return Enumerable.Empty<string>();
+
+            var normalizePaths = new HashSet<string>();
+            foreach (var locationPath in LocationPaths)
+            {
+                if (Path.IsPathRooted(locationPath))
```

Comment:
Yeah, you're right that will be counter intuitive for the global config file case. Maybe a doc comment on `LocationPaths` to help describe the expected values?